sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -70,7 +70,7 @@ export interface DetectLanguageSuccessResult extends TextAnalyticsSuccessResult
 }
 
 // @public
-export type DocumentSentimentValue = 'positive' | 'neutral' | 'negative' | 'mixed';
+export type DocumentSentimentValue = "positive" | "neutral" | "negative" | "mixed";
```

Comment:
Not sure why this file shows changes at all, may be because it was not run through prettier...